learning/features/math/vector_math.rst

```diff
@@ -123,7 +123,7 @@ Let's look at two common uses for vector addition and subtraction.
 
 - Movement
 
-A vector can represent **any** quantity with a magnitude and direction. In
+A vector can represent **any** quantity with a magnitude and direction. Typical examples are: position, direction, velocity, acceleration, force etc. In
```

Comment:
I agree that adding examples here is helpful, but I think that the wording can be improved. Having direction listed as an example when we just defined a vector as "magnitude and direction" may be confusing. In addition, I think that the "etc." at the end is not necessary, as this has already been established to be a limited list of examples.

I would probably write it like so: "Typical examples are position, velocity, acceleration, and force."

Reply:
> ...I wanted to suggest some kind of a comprehensive use case...

I don't agree that the list should try to be comprehensive in any way, it just needs to give an idea of what they are and how they are useful. What you have does that pretty well in my opinion, it just needs to be cleaned up a bit.

> Would it help if we say "Typical use cases..."?

I would say that changing the beginning to "Typical use cases include..." can work too. 

> Or maybe move the list in another part of the document?

I think the placement in the doc is appropriate as it is.

Reply:
Agreed, there are too many use cases to be comprehensive. Remember the audience here. This doc typically isn't going to be read by users as their entry point to Godot. They're going to wind up here when they hit their first Vector2 and don't know what that is.